src/qtism/data/storage/xml/marshalling/OutcomeDeclarationMarshaller.php

```diff
@@ -151,12 +171,20 @@ protected function unmarshall(DOMElement $element)
             }
 
             // deal with normalMinimum.
-            if (Version::compare($version, '2.1.0', '>=') === true && ($normalMinimum = $this->getDOMElementAttributeAs($element, 'normalMinimum', 'float')) !== null) {
+            if (Version::compare($version, '2.1.0', '>=') === true && ($normalMinimum = $this->getDOMElementAttributeAs(
+                    $element,
+                    'normalMinimum',
+                    'float'
+                )) !== null) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place